### PR TITLE
Sauce testing

### PR DESCRIPTION
**Pull request by JostCrow**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -19,12 +19,32 @@ webpackConfig.externals = [];
 
 // The main configuration
 module.exports = function(config) {
-    config.set({
-        browserStack: {
-            username: '',
-            accessKey: ''
+    if (!process.env.SAUCE_USERNAME || !process.env.SAUCE_ACCESS_KEY) {
```
Comment (svenvandescheur) [inline karma.conf.js]: Do we want to run saucelabs even on local? I'd suggest detecting process.env.CI i.c.w. a setting to check if we're on Travis (https://docs.travis-ci.com/user/environment-variables/#Convenience-Variables) or explicitly say we want to test against Sauce Labs.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "karma-jasmine": "^1.0.2",
     "karma-jasmine-ajax": "^0.1.13",
     "karma-jasmine-jquery": "^0.1.1",
+    "karma-sauce-launcher": "^1.1.0",
```
Comment (svenvandescheur) [inline package.json]: Don't forget to remove karma-browserstack-launcher.

Comment (JostCrow) [inline package.json]: It is removed!

**Comment by svenvandescheur** [comment]
LGTM, but why are the tests failing?

**Comment by JostCrow** [comment]
https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-BrowserName

**Pull request merged**